README.md

```diff
@@ -196,6 +196,16 @@ Gets the global dispatcher used by Common API Methods.
 
 Returns: `Dispatcher`
 
+### `UrlObject`
+
+* **port** `string | number | null` (optional)
+* **path** `string | null` (optional)
+* **pathname** `string | null` (optional)
+* **hostname** `string | null` (optional)
+* **origin** `string | null` (optional)
+* **protocol** `string | null` (optional)
+* **search** `string | null` (optional)
```

Comment:
We don't support all of these properties. Please see lib/core/util/parseURL

Reply:
I got the list from exactly [parseURL](https://github.com/nodejs/undici/blob/0b4694722d17c279e0e960b70046ea54654e254d/lib/core/util.js#L21). Which property should not be here in your opinion?